src/concepts/segments.md

```diff
@@ -25,10 +25,14 @@ so that your communication is forwards compatible.
 
 ### Regular users v3
 
+`clients_last_seen.is_regular_user_v3`
```

Comment:
I don't feel too strongly, but it's possibly non-obvious that this is <tablename>.<fieldname>  Could spell it out "This segment is available in the X field of the Y table" if you like.

Reply:
I think that as we add more segments, the extra words might become tiresome? So I'll leave it for now